### PR TITLE
Update conda packaging in dev

### PR DESCRIPTION
**Pull request by folmos-at-orange**
Same as #214  but for `dev`

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,7 +20,11 @@ CMakeFiles/
 # Test exceptions
 test/UnitTests/*/results/
 test/LearningTest/*/*/*/results/
+test/*/results/
+test/LearningTest/*/*/*/results.ref/time.log
+results/
```
Comment (bruno-at-orange) [inline .gitignore]: I think line 23 and 25 are redundant with 21 and 22. We can choose to set only line 25 over 21 22 and 23 (to discuss)
line 24 is wrong, we have to keep time.log in results.ref

**Review by bruno-at-orange — APPROVED** [review]
LGTM
just modify the .gitignore file

File: .gitignore (hunk 1 above)
Comment (folmos-at-orange) [reply .gitignore]: Ok

Comment (folmos-at-orange) [reply .gitignore]: Just not to have a range too large how about `test/**/results/` ?

**Pull request merged**